### PR TITLE
Add "All Countries" option to create and edit form

### LINKED ISSUE #40: Add Select All for Countries on Resource Edit
Gotta add them all!

### PR DESCRIPTION
**Pull request by maxkadel**
Closes #40

For an example, see https://dss-staging.princeton.edu/resources/4894

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -57,9 +57,10 @@
...
   # config.filter_gems_from_backtrace("gem name")
-
-  config.include Devise::Test::ControllerHelpers, type: :controller
-  config.include Devise::Test::ControllerHelpers, type: :view
-  config.include Devise::Test::IntegrationHelpers, type: :request
-  config.include Devise::Test::IntegrationHelpers, type: :feature
+  [:controller, :view].each do |type|
+    config.include Devise::Test::ControllerHelpers, type:
+  end
+  [:feature, :request, :system].each do |type|
```
Comment (sandbergja) [inline spec/rails_helper.rb:63]: Nice!  I don't think we have any feature specs in this project, though.  Could you remove `:feature` while you're at it?
```suggestion
  [:request, :system].each do |type|
```

File: app/controllers/resources_controller.rb [hunk 2]
```diff
@@ -30,7 +30,7 @@ def edit
   # POST /resources.json
   def create
     @resource = Resource.new(resource_params)
-
+
```
Comment (sandbergja) [inline app/controllers/resources_controller.rb:33]: Could you remove these spaces please?

**Review by sandbergja — CHANGES_REQUESTED** [review]
Nice!  The UI looks really slick.  Just two smol things.

**Comment by kevinreiss** [comment]
I looked at this in staging and agree with @sandbergja this looks good and I think will definitely address the pain point the maintainers have had managing the country lists.

File: app/controllers/resources_controller.rb (hunk 2 above)
Comment (maxkadel) [reply app/controllers/resources_controller.rb]: Done!

File: spec/rails_helper.rb (hunk 1 above)
Comment (maxkadel) [reply spec/rails_helper.rb]: Done!

**Review by sandbergja — APPROVED** [review]
Thanks, @maxkadel !

**Pull request merged**